ext/storage_sources/azure_store/azure_storage_source.cpp

```diff
@@ -433,7 +433,7 @@ azure_file_system_exists(
       "azure_file_system_exists: Checking object: " + std::string(name) + " exists in Azure.");
     // Check whether the object exists in the cloud.
     int ret;
-    if (ret = azure_fs->azure_conn->object_exists(name, *existp, size) != 0) {
+    if ((ret = azure_fs->azure_conn->object_exists(name, *existp, size)) != 0) {
```

Comment:
typo: too many brackets

Reply:
I needed this to compile with clang under release mode.

Reply:
Ah yep, I missed the `=`. Resolved